cmd/fluxd/main.go

```diff
@@ -155,6 +155,10 @@ func main() {
 		sshKeyType   = optionalVar(fs, &ssh.KeyTypeValue{}, "ssh-keygen-type", "-t argument to ssh-keygen (default unspecified)")
 		sshKeygenDir = fs.String("ssh-keygen-dir", "", "directory, ideally on a tmpfs volume, in which to generate new SSH keys when necessary")
 
+		// manifest regeration
+		// TODO(fons): make false by default before merging
+		manifestGeneration = fs.Bool("manifest-generation", true, "experimental; search for .flux.yaml files for manifest generation")
```

Comment:
This is intentionally `true` by default until/if we decide to merge to simplify testing by the community